connext_cmake_module/cmake/Modules/FindConnext.cmake

```diff
@@ -211,29 +209,38 @@ if(NOT "${_NDDSHOME} " STREQUAL " ")
   endif()
 
   set(Connext_LIBRARIES "")
+  # first create an imported target for each Connext library
+  foreach(_base_name IN LISTS _base_names)
+    add_library(${_base_name} UNKNOWN IMPORTED)
+    list(APPEND Connext_LIBRARIES ${_base_name})
+  endforeach()
+  list(LENGTH _base_names _base_names_len)
+  math(EXPR _loop_iterations "${_base_names_len} - 1")
   if(_found_all_optimized_libraries)
-    foreach(_lib IN LISTS _optimized_libraries)
-      # if libraries of both types are found use prepend the build configuration
-      if(_found_all_debug_libraries)
-        list(APPEND Connext_LIBRARIES "optimized")
-      endif()
-      list(APPEND Connext_LIBRARIES "${_lib_path}/${_lib}")
+    # for each of the optimized libraries set the IMPORTED_LOCATION for the RELEASE config
+    foreach(_val RANGE ${_loop_iterations})
+      list(GET _base_names ${_val} _base_name)
+      list(GET _optimized_libraries ${_val} _lib)
+      set_target_properties(${_base_name} PROPERTIES IMPORTED_LOCATION_RELEASE "${_lib_path}/${_lib}")
```

Comment:
What about the location for other release-like configurations like `RelWithDebInfo`?